assemblers/org.osbuild.oci-archive

```diff
@@ -150,24 +147,24 @@ def blobs_add_layer(blobs: str, tree: str):
         "--no-selinux",
         "--acls",
         "--xattrs",
-        "-cf", layer_file,
-        "-C", tree,
+        "-cf",
```

Comment:
There are too many changes here to review them all :) I like most of what I am seeing, so I'm pointing out a couple places I think this makes it less readable. In this case it has disconnected cmdline args from their values.

Reply:
This is the one brought up previously as mostly the deal-breaker for using auto formatting.